### PR TITLE
Creates new Python Package 

### LINKED ISSUE #55: Refactor ils-middleware to be installed as a package
Allows for better import patterns (able to use absolute imports in code and tests) and not require developer or system to set  PYTHONPATH for code inclusion.

### PR DESCRIPTION
**Pull request by jermnelson**
Fixes #55. 

Adds new `ils_middleware` package that includes `dags` and `tasks` at the same level and uses absolute imports in all source code.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,4 +4,6 @@ USER root
 RUN apt-get -y update && apt-get -y install git
 USER airflow
 
-COPY --chown=airflow:root ./dags /opt/airflow/dags
\ No newline at end of file
+COPY --chown=airflow:root dist/ils_middleware-0.1.0-py3-none-any.whl .
```
Comment (jermnelson) [inline Dockerfile:7]: I think there should be a better method to install `ils_middleware` in the image. I need to do some more investigation.

**Review by jmartin-sul — CHANGES_REQUESTED** [review]
noticed what seemed like a small mistaken name change for a parameter, and then another thing that might be an unused variable (or might just be me not understanding an unfamiliar construct)

File: ils_middleware/dags/cornell.py [hunk 2]
```diff
@@ -48,9 +48,12 @@
     )
     connect_okapi_cmd = """exit 0"""
 
+    folio_login = FolioLogin(tenant="cornell", username="", password="")
+
     send_to_folio = FolioRequest(
+        task_id="cornell_send_to_folio",
         tenant="cornell",
-        token=FolioLogin(tenant="cornell", username="", password=""),
+        session_token="{{ task_instance.xcom_pull(key='return_value', task_ids=['folio_login'])[0]}}",
```
[the remark this answers is not in the document]
Comment (jermnelson) [reply ils_middleware/dags/cornell.py]: Yes, good catch! Now changed to use `token`.

File: ils_middleware/dags/cornell.py [hunk 3]
```diff
@@ -48,9 +48,12 @@
     )
     connect_okapi_cmd = """exit 0"""
 
+    folio_login = FolioLogin(tenant="cornell", username="", password="")
```
[the remark this answers is not in the document]
Comment (jermnelson) [reply ils_middleware/dags/cornell.py:51]: The `folio_login` task saves the FOLIO JWT to the default `return_value` in inter-tasks communication (XCOM) that is passed to the next task with the `task_ids=['folio_login']` with the call in 56. The default task_id for `FolioLogin` is `folio_login`.

[the remark this answers is not in the document]
Comment (jmartin-sul) [reply ils_middleware/dags/cornell.py:51]: sorta/kinda understood that, but either way, convinced that it isn't cruft or an oversight 🙂

**Review by jmartin-sul — APPROVED** [review]

**Pull request merged**